### ISSUE TITLE
WHAMR!: sep_reverb_noisy 1dB lower than the reported performance

### ISSUE THREAD

**Issue by hangtingchen** [mpariente/asteroid#177]
#### What is your question?

Hi, 
I am trying task of both separation and denoise on WHAMR!, but my SI-SDR is about 3.94dB, 1dB lower than that in README. 
Could you please upload the log so that I can check this by myself ?

Thanks a lot !

**Comment by mpariente** [comment]
Can you add you're training config please? I'll look for the log today.

**Comment by hangtingchen** [comment]
Thanks ! This is my config:
data: 
  mode: min
  nondefault_nsrc: null
  sample_rate: 8000
  task: sep_reverb_noisy
  train dir: data/wav8k/min/tr
  valid dir: data/wav8k/min/cv
filterbank: 
  kernel size: 40
  n_filters: 512
  stride: 20
main_args: 
  exp_dir: exp/train_tasnet_sep_reverb_noisy_8kmin_db50b168
  help: null
masknet: 
  dropout: 0.3
  n_layers: 4
  n_src: 2
  n_units: 600
optim: 
  lr: 0.001
  optimizer: adam
  weight_decay: 1.0e-05
training: 
  batch size: 32
  early_stoping: true
  epochs: 200
  half_lr: true
  num_workers: 16

My final loss is about -3.98 (SI-SNR)

**Comment by mpariente** [comment]
Here are the log files : [logs.tar.gz](https://github.com/mpariente/asteroid/files/4965670/logs.tar.gz)

Here is the conf


```yaml
data:
  mode: min
  nondefault_nsrc: null
  sample_rate: 8000
  task: sep_reverb_noisy
  train_dir: data/wav8k/min/tr
  valid_dir: data/wav8k/min/cv
filterbank:
  kernel_size: 40
  n_filters: 512
  stride: 20
main_args:
  exp_dir: exp/train_tasnet_sep_reverb_noisy_plain_pl0.6/
  gpus: '-1'
  help: null
masknet:
  dropout: 0.3
  n_layers: 4
  n_src: 2
  n_units: 600
optim:
  lr: 0.001
  optimizer: adam
  weight_decay: 1.0e-05
positional arguments: {}
training:
  batch_size: 32
  early_stop: true
  epochs: 200
  half_lr: true
  num_workers: 4
```

In pytorch lightning 0.6.0, early stopping was broken so training didn't stop and the performance improved. Could you also try without early stopping to see if this is better please? 
I need to make the patience higher IMO

**Comment by hangtingchen** [comment]
The log is similar with mine. I think I can reproduce the results with more training epochs.
Thanks for your patience !

**Comment by mpariente** [comment]
Could you share your log here please? I'm curious when the early stopping happens

**Comment by hangtingchen** [comment]
Sure. I will download the log from the lab tomorrow

**Comment by hangtingchen** [comment]
Sorry for late reply. This is my log.
[train_tasnet_sep_reverb_noisy_8kmin_db50b168.tar.gz](https://github.com/mpariente/asteroid/files/4975115/train_tasnet_sep_reverb_noisy_8kmin_db50b168.tar.gz)
